src/Authenticator/ResultInterface.php

```diff
@@ -54,7 +54,7 @@ public function isValid();
     /**
      * Get the result code for this authentication attempt.
      *
-     * @return int
+     * @return string
      */
     public function getCode();
```

Comment:
Should we change the method name to reflect this change? This is no longer a “code”, more like “status”?

Reply:
getStatus() or getStatusCode()?
It kind of is still a code, just not an int but string one.

Reply:
getStatus() feels more appropriate.